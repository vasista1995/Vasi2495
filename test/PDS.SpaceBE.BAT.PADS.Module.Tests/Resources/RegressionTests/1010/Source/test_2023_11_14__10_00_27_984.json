{
  "IdSource": "SPACEACT2:BAT:BE:RU320233B01:BE1I:Wire_Bond1:WirePull:57747:240170467:16",
  "Data_flat_Metadata": {
    "Device": "S8803B",
    "ParameterClass": "PPAP",
    "CFComment": "FOL",
    "Segment": "WB",
    "FourDReport": "*",
    "ParameterName": "WirePull",
    "ParameterUnit": "g",
    "ParameterOper": "Wire_Bond1",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "ChannelId": "57747",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-11-14T02:12:30Z",
    "CreatedTimestamp": "2023-11-14T09:12:30Z",
    "UpdatedTimestamp": "2023-11-14T14:12:22Z",
    "UpdatedTimestampUtc": "2023-11-14T07:12:22Z",
    "ExportedTimestampUtc": "2023-11-14T09:12:30Z",
    "ExportedTimestamp": "2023-11-14T09:12:30Z",
    "SampleTimestampUtc": "2023-11-14T02:12:30Z",
    "SampleTimestamp": "2023-11-14T09:12:30Z",
    "SourceDataLevel": "C",
    "ChannelName": "Wire_PullForce/CU/50um/PDS36/CU6/LSS",
    "ChannelDescr": "LSS",
    "RvStoreFlag": "Y",
    "CalcCreated": "09/14/2022 17:44:56",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": "iscw8cc2082mw1",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 1876374,
    "ViolCount": 1,
    "CalculationStrategy": "ND",
    "LdsID": "1010",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "16",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "N",
        "SpecLowEnabled": "Y",
        "SpecLow": 10.0,
        "SpecTarget": 20.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYYYNNYNYYNNNNNNNNNNNNNNNNNNYNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "41,7658",
        "Median": "41,387",
        "Sigma": "3,83609797841505",
        "Min": "36,497",
        "Max": "46,522",
        "Q1": "40,219",
        "Q3": "44,204"
      },
      "ControlLimits": {
        "ctrl_mean_low": 36.38535170871,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 44.881127046131,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 53.3769023835519,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "Y",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "ctrl_sigma_low": 0.244502118600319,
        "CtrlSigmaLowEnabled": "Y",
        "CtrlSigmaHighEnabled": "N",
        "ctrl_range_low": 0.687968768320747,
        "CtrlRangeLowEnabled": "Y",
        "ctrl_range_target": 4.11534835713174,
        "ctrl_range_high": 13.969358381041,
        "CtrlRangeHighEnabled": "Y",
        "ExtMaMV": 42.489775,
        "ExtEwmaMV": 41.8662500458282,
        "ExtMsMV": 0.873135453409149,
        "EwmaS": 2.54372712776545,
        "EwmaR": 6.49309894552272,
        "ExtEwmaMVLCL": 42.0488602483523,
        "ExtEwmaMVCenter": 44.8812908603751,
        "ExtEwmaMVUCL": 47.7137214723979,
        "ExtMaMVLCL": 42.8784599673734,
        "ExtMaMVCenter": 44.8812908603751,
        "ExtMaMVUCL": 46.8841217533768,
        "EwmaSLCL": 0.53443043510989,
        "EwmaSCenter": 1.80653891724822,
        "EwmaSUCL": 3.07864739938655,
        "EwmaRLCL": 1.34561613438603,
        "EwmaRCenter": 4.41631038332714,
        "EwmaRUCL": 7.48700463226825,
        "ExtMSMVLCL": 0.0029099179771159,
        "ExtMSMVCenter": 1.3722299528235,
        "ExtMSMVUCL": 5.51230086243606,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 33.1914464381814,
        "RawUal": 56.5711352825689,
        "SigmaLal": -1.18666368715626,
        "SigmaUal": 10.9039655257908,
        "RangeLal": -2.73941082049025,
        "RangeUal": 23.8233684049503
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 240170467,
        "Seqnr": 2,
        "Value": 41.387,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "r/c : wire feed correction too low\nc/a : inc wire feed correction from 0 to 20\nagus\n--- 2023-11-14 14:12:22 iscw8cc2082mw1 ---\nRC : WIRE FEED CORRECTION TOO LOW\nCA : ADJUST WIRE FEED CORRECTION 300 - 330\n\nOKI",
        "ViolationList": "10",
        "PrimaryViolation": "10",
        "PrimaryViolationComments": "r/c : wire feed correction too low\nc/a : inc wire feed correction from 0 to 20\nagus\n--- 2023-11-14 14:12:22 iscw8cc2082mw1 ---\nRC : WIRE FEED CORRECTION TOO LOW\nCA : ADJUST WIRE FEED CORRECTION 300 - 330\n\nOKI"
      },
      {
        "SampleId": 240170467,
        "Seqnr": 1,
        "Value": 46.522,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "r/c : wire feed correction too low\nc/a : inc wire feed correction from 0 to 20\nagus\n--- 2023-11-14 14:12:22 iscw8cc2082mw1 ---\nRC : WIRE FEED CORRECTION TOO LOW\nCA : ADJUST WIRE FEED CORRECTION 300 - 330\n\nOKI",
        "ViolationList": "10",
        "PrimaryViolation": "10",
        "PrimaryViolationComments": "r/c : wire feed correction too low\nc/a : inc wire feed correction from 0 to 20\nagus\n--- 2023-11-14 14:12:22 iscw8cc2082mw1 ---\nRC : WIRE FEED CORRECTION TOO LOW\nCA : ADJUST WIRE FEED CORRECTION 300 - 330\n\nOKI"
      },
      {
        "SampleId": 240170467,
        "Seqnr": 4,
        "Value": 40.219,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "r/c : wire feed correction too low\nc/a : inc wire feed correction from 0 to 20\nagus\n--- 2023-11-14 14:12:22 iscw8cc2082mw1 ---\nRC : WIRE FEED CORRECTION TOO LOW\nCA : ADJUST WIRE FEED CORRECTION 300 - 330\n\nOKI",
        "ViolationList": "10",
        "PrimaryViolation": "10",
        "PrimaryViolationComments": "r/c : wire feed correction too low\nc/a : inc wire feed correction from 0 to 20\nagus\n--- 2023-11-14 14:12:22 iscw8cc2082mw1 ---\nRC : WIRE FEED CORRECTION TOO LOW\nCA : ADJUST WIRE FEED CORRECTION 300 - 330\n\nOKI"
      },
      {
        "SampleId": 240170467,
        "Seqnr": 3,
        "Value": 44.204,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "r/c : wire feed correction too low\nc/a : inc wire feed correction from 0 to 20\nagus\n--- 2023-11-14 14:12:22 iscw8cc2082mw1 ---\nRC : WIRE FEED CORRECTION TOO LOW\nCA : ADJUST WIRE FEED CORRECTION 300 - 330\n\nOKI",
        "ViolationList": "10",
        "PrimaryViolation": "10",
        "PrimaryViolationComments": "r/c : wire feed correction too low\nc/a : inc wire feed correction from 0 to 20\nagus\n--- 2023-11-14 14:12:22 iscw8cc2082mw1 ---\nRC : WIRE FEED CORRECTION TOO LOW\nCA : ADJUST WIRE FEED CORRECTION 300 - 330\n\nOKI"
      },
      {
        "SampleId": 240170467,
        "Seqnr": 5,
        "Value": 36.497,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 1,
        "ViolationComments": "r/c : wire feed correction too low\nc/a : inc wire feed correction from 0 to 20\nagus\n--- 2023-11-14 14:12:22 iscw8cc2082mw1 ---\nRC : WIRE FEED CORRECTION TOO LOW\nCA : ADJUST WIRE FEED CORRECTION 300 - 330\n\nOKI",
        "ViolationList": "10",
        "PrimaryViolation": "10",
        "PrimaryViolationComments": "r/c : wire feed correction too low\nc/a : inc wire feed correction from 0 to 20\nagus\n--- 2023-11-14 14:12:22 iscw8cc2082mw1 ---\nRC : WIRE FEED CORRECTION TOO LOW\nCA : ADJUST WIRE FEED CORRECTION 300 - 330\n\nOKI"
      }
    ],
    "SampleSize": 5
  },
  "ProductionAction": {
    "Id": "SPACEACT1:BAT:BE:BE1I:Wire_Bond1:WirePull:57747",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "BAT:BE:RU320233B01:BE1I:Wire_Bond1:WirePull:57747:240170467:16"
  }
}