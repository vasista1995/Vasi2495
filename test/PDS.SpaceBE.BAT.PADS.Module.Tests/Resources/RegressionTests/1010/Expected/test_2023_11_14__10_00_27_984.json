{
  "Data_flat_Metadata": {
    "ParameterOper": "Wire_Bond1",
    "ParameterFacility": "BE1I",
    "SiteKey": "BAT",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-11-14T02:12:30Z",
    "BeginTimestamp": "2023-11-14T09:12:30Z",
    "EndTimestampUtc": "2023-11-14T02:12:30Z",
    "EndTimestamp": "2023-11-14T09:12:30Z"
  },
  "Data_flat_StrangeMetadata": {
    "Device": "S8803B",
    "ParameterClass": "PPAP",
    "Segment": "WB"
  },
  "Data_1list_Parameters": [
    {
      "Data_1list_RawValues": [
        {
          "Seqnr": 2,
          "SampleId": 240170467,
          "Value": 41.387,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T09:12:30Z",
          "SampleTimestampUtc": "2023-11-14T02:12:30Z",
          "CreatedTimestamp": "2023-11-14T09:12:30Z",
          "CreatedTimestampUtc": "2023-11-14T02:12:30Z",
          "UpdatedTimestamp": "2023-11-14T14:12:22Z",
          "UpdatedTimestampUtc": "2023-11-14T07:12:22Z",
          "NumViolations": 1,
          "ViolationComments": "r c : wire feed correction too low c a : inc wire feed correction from 0 to 20 agus --- 2023-11-14 14:12:22 iscw8cc2082mw1 --- RC : WIRE FEED CORRECTION TOO LOW CA : ADJUST WIRE FEED CORRECTION 300 - 330  OKI",
          "ViolationList": "10",
          "PrimaryViolation": "10",
          "PrimaryViolationComments": "r c : wire feed correction too low c a : inc wire feed correction from 0 to 20 agus --- 2023-11-14 14:12:22 iscw8cc2082mw1 --- RC : WIRE FEED CORRECTION TOO LOW CA : ADJUST WIRE FEED CORRECTION 300 - 330  OKI",
          "SampleMean": 41.7658,
          "SampleMin": 36.497,
          "SampleMax": 46.522,
          "SampleMedian": 41.387,
          "SampleStdev": 3.83609797841505,
          "SampleQ1": 40.219,
          "SampleQ3": 44.204,
          "SampleSize": 5
        },
        {
          "Seqnr": 1,
          "SampleId": 240170467,
          "Value": 46.522,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T09:12:30Z",
          "SampleTimestampUtc": "2023-11-14T02:12:30Z",
          "CreatedTimestamp": "2023-11-14T09:12:30Z",
          "CreatedTimestampUtc": "2023-11-14T02:12:30Z",
          "UpdatedTimestamp": "2023-11-14T14:12:22Z",
          "UpdatedTimestampUtc": "2023-11-14T07:12:22Z",
          "NumViolations": 1,
          "ViolationComments": "r c : wire feed correction too low c a : inc wire feed correction from 0 to 20 agus --- 2023-11-14 14:12:22 iscw8cc2082mw1 --- RC : WIRE FEED CORRECTION TOO LOW CA : ADJUST WIRE FEED CORRECTION 300 - 330  OKI",
          "ViolationList": "10",
          "PrimaryViolation": "10",
          "PrimaryViolationComments": "r c : wire feed correction too low c a : inc wire feed correction from 0 to 20 agus --- 2023-11-14 14:12:22 iscw8cc2082mw1 --- RC : WIRE FEED CORRECTION TOO LOW CA : ADJUST WIRE FEED CORRECTION 300 - 330  OKI",
          "SampleMean": 41.7658,
          "SampleMin": 36.497,
          "SampleMax": 46.522,
          "SampleMedian": 41.387,
          "SampleStdev": 3.83609797841505,
          "SampleQ1": 40.219,
          "SampleQ3": 44.204,
          "SampleSize": 5
        },
        {
          "Seqnr": 4,
          "SampleId": 240170467,
          "Value": 40.219,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T09:12:30Z",
          "SampleTimestampUtc": "2023-11-14T02:12:30Z",
          "CreatedTimestamp": "2023-11-14T09:12:30Z",
          "CreatedTimestampUtc": "2023-11-14T02:12:30Z",
          "UpdatedTimestamp": "2023-11-14T14:12:22Z",
          "UpdatedTimestampUtc": "2023-11-14T07:12:22Z",
          "NumViolations": 1,
          "ViolationComments": "r c : wire feed correction too low c a : inc wire feed correction from 0 to 20 agus --- 2023-11-14 14:12:22 iscw8cc2082mw1 --- RC : WIRE FEED CORRECTION TOO LOW CA : ADJUST WIRE FEED CORRECTION 300 - 330  OKI",
          "ViolationList": "10",
          "PrimaryViolation": "10",
          "PrimaryViolationComments": "r c : wire feed correction too low c a : inc wire feed correction from 0 to 20 agus --- 2023-11-14 14:12:22 iscw8cc2082mw1 --- RC : WIRE FEED CORRECTION TOO LOW CA : ADJUST WIRE FEED CORRECTION 300 - 330  OKI",
          "SampleMean": 41.7658,
          "SampleMin": 36.497,
          "SampleMax": 46.522,
          "SampleMedian": 41.387,
          "SampleStdev": 3.83609797841505,
          "SampleQ1": 40.219,
          "SampleQ3": 44.204,
          "SampleSize": 5
        },
        {
          "Seqnr": 3,
          "SampleId": 240170467,
          "Value": 44.204,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T09:12:30Z",
          "SampleTimestampUtc": "2023-11-14T02:12:30Z",
          "CreatedTimestamp": "2023-11-14T09:12:30Z",
          "CreatedTimestampUtc": "2023-11-14T02:12:30Z",
          "UpdatedTimestamp": "2023-11-14T14:12:22Z",
          "UpdatedTimestampUtc": "2023-11-14T07:12:22Z",
          "NumViolations": 1,
          "ViolationComments": "r c : wire feed correction too low c a : inc wire feed correction from 0 to 20 agus --- 2023-11-14 14:12:22 iscw8cc2082mw1 --- RC : WIRE FEED CORRECTION TOO LOW CA : ADJUST WIRE FEED CORRECTION 300 - 330  OKI",
          "ViolationList": "10",
          "PrimaryViolation": "10",
          "PrimaryViolationComments": "r c : wire feed correction too low c a : inc wire feed correction from 0 to 20 agus --- 2023-11-14 14:12:22 iscw8cc2082mw1 --- RC : WIRE FEED CORRECTION TOO LOW CA : ADJUST WIRE FEED CORRECTION 300 - 330  OKI",
          "SampleMean": 41.7658,
          "SampleMin": 36.497,
          "SampleMax": 46.522,
          "SampleMedian": 41.387,
          "SampleStdev": 3.83609797841505,
          "SampleQ1": 40.219,
          "SampleQ3": 44.204,
          "SampleSize": 5
        },
        {
          "Seqnr": 5,
          "SampleId": 240170467,
          "Value": 36.497,
          "is_flagged": "N",
          "SampleTimestamp": "2023-11-14T09:12:30Z",
          "SampleTimestampUtc": "2023-11-14T02:12:30Z",
          "CreatedTimestamp": "2023-11-14T09:12:30Z",
          "CreatedTimestampUtc": "2023-11-14T02:12:30Z",
          "UpdatedTimestamp": "2023-11-14T14:12:22Z",
          "UpdatedTimestampUtc": "2023-11-14T07:12:22Z",
          "NumViolations": 1,
          "ViolationComments": "r c : wire feed correction too low c a : inc wire feed correction from 0 to 20 agus --- 2023-11-14 14:12:22 iscw8cc2082mw1 --- RC : WIRE FEED CORRECTION TOO LOW CA : ADJUST WIRE FEED CORRECTION 300 - 330  OKI",
          "ViolationList": "10",
          "PrimaryViolation": "10",
          "PrimaryViolationComments": "r c : wire feed correction too low c a : inc wire feed correction from 0 to 20 agus --- 2023-11-14 14:12:22 iscw8cc2082mw1 --- RC : WIRE FEED CORRECTION TOO LOW CA : ADJUST WIRE FEED CORRECTION 300 - 330  OKI",
          "SampleMean": 41.7658,
          "SampleMin": 36.497,
          "SampleMax": 46.522,
          "SampleMedian": 41.387,
          "SampleStdev": 3.83609797841505,
          "SampleQ1": 40.219,
          "SampleQ3": 44.204,
          "SampleSize": 5
        }
      ],
      "CFComment": "FOL",
      "FourDReport": "*",
      "Data_flat_Limits": {
        "CKC_ID": "16",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "N",
          "SpecLowEnabled": "Y",
          "spec_low": 10.0,
          "target": 20.0
        },
        "ControlLimits": {
          "ctrl_mean_low": 36.38535170871,
          "CtrlMeanLowEnabled": "Y",
          "ctrl_mean_target": 44.881127046131,
          "CtrlMeanTargetEnabled": "Y",
          "ctrl_mean_high": 53.3769023835519,
          "CtrlMeanHighEnabled": "Y",
          "CtrlLowEnabled": "N",
          "CtrlHighEnabled": "N",
          "ctrl_stdev_low": 0.244502118600319,
          "CtrlStdevLowEnabled": "Y",
          "CtrlSigmaCenterEnabled": "N",
          "CtrlRangeCenterEnabled": "Y",
          "CtrlCenterEnabled": "N",
          "CtrlStdevHighEnabled": "N",
          "ctrl_range_low": 0.687968768320747,
          "CtrlRangeLowEnabled": "Y",
          "ctrl_range_target": 4.11534835713174,
          "ctrl_range_high": 13.969358381041,
          "CtrlRangeHighEnabled": "Y"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 5,
        "base_count": 5,
        "flagged_count": 0,
        "mean": 41.7658,
        "median": 41.387,
        "stdev": 3.431110339234225,
        "range": 10.024999999999999,
        "min": 36.497,
        "max": 46.522,
        "q02": 36.794760000000004,
        "q05": 37.2414,
        "q25": 40.219,
        "q75": 44.204,
        "q95": 46.0584,
        "q98": 46.33656,
        "NumViolations": 1,
        "ViolationComments": "r c : wire feed correction too low c a : inc wire feed correction from 0 to 20 agus --- 2023-11-14 14:12:22 iscw8cc2082mw1 --- RC : WIRE FEED CORRECTION TOO LOW CA : ADJUST WIRE FEED CORRECTION 300 - 330  OKI",
        "ViolationList": "10",
        "PrimaryViolation": "10",
        "PrimaryViolationComments": "r c : wire feed correction too low c a : inc wire feed correction from 0 to 20 agus --- 2023-11-14 14:12:22 iscw8cc2082mw1 --- RC : WIRE FEED CORRECTION TOO LOW CA : ADJUST WIRE FEED CORRECTION 300 - 330  OKI",
        "Samples": "240170467"
      },
      "ParameterName": "WirePull",
      "ParameterUnit": "g",
      "ChannelId": "57747",
      "SourceDataLevel": "C",
      "channel_name": "Wire_PullForce CU 50um PDS36 CU6 LSS",
      "channel_description": "LSS",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-11-14T02:12:30Z",
      "created_timestamp": "2023-11-14T09:12:30Z",
      "updated_timestamp_utc": "2023-11-14T07:12:22Z",
      "updated_timestamp": "2023-11-14T14:12:22Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-11-14T07:12:22Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED2:BAT:BE:BE1I:Wire_Bond1:",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "Measlot:",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "UniqueSourceKey": "BAT:BE:RU320233B01:BE1I:Wire_Bond1:WirePull:57747:240170467:16",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-11-14T09:12:30Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "Measlot::SPACEAGGED2:BAT:BE:BE1I:Wire_Bond1::ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "BAT"
  }
}